pipeline/tests/pil.rs

```diff
@@ -306,6 +306,13 @@ fn naive_byte_decomposition_gl() {
     verify_pil(f, Default::default());
 }
 
+#[test]
+#[should_panic = "NoVariableDegreeAvailable"]
```

Comment:
Does this at least run witgen?

Reply:
Yes! The error is thrown by the backend.